framework/test/unit/modules/token/cc_method.spec.ts

```diff
@@ -360,7 +409,7 @@ describe('TokenInteroperableMethod', () => {
 			);
 		});
 
-		it('should deduct escrow account for fee and credit to receiving chain escrow account if ccm command is not transfer', async () => {
+		it('should deduct sending chain escrow account for fee and credit to receving chain escrow account if ccm failed', async () => {
```

Comment:
The test case is validating failed CCM case, and verifying if relayerAddress is credited, perhaps can be rephrased to, `should deduct sending chain escrow account for fee and credit to relayer account if ccm failed`.

Reply:
I've updated the logic to use ccmFailed, test description and updated comment in the source code to make it more clear. Same for below comment